src/toolchain.rs

```diff
@@ -556,6 +557,90 @@ impl<'a> Toolchain<'a> {
         }
     }
 
+    fn get_component_suggestion(
+        &self,
+        component: &Component,
+        manifest: &Manifest,
+        only_instaled: bool,
+    ) -> String {
```

Comment:
This ought to return `Option<String>` in case there isn't a suggestion to be made.